docs/source/cluster_management/backup-restore.rst

```diff
@@ -1,106 +0,0 @@
-.. _backup-restore:
-
-==================
-Backup and Restore
-==================
-
-Overview
-========
-
-Backup and restore for AtlasDB are non-trivial operations that are not satisfied by simply backing up and restoring the underyling key value store you are using to store your data in.  This page describes how to take advantage of built in AtlasDB utilities, namely the Atlas CLI, in order to perform a live consistent point in time backup and subsequent restore of that backup from total data loss.
```

Comment:
should probably have a follow up to actually preserve some of the information on how one would perform a backup.